ui/pages/Project/components/FamilyPage.jsx

```diff
@@ -201,16 +203,59 @@ const mapStateToProps = (state, ownProps) => ({
 
 export const FamilyDetail = connect(mapStateToProps)(BaseFamilyDetail)
 
-const FamilyPage = ({ match }) => (
+const MyFamilyPage = ({ match }) => (
```

Comment:
keep this named the same, rename the new component to `FamilyPageRouter` as its more descritive